### PR TITLE
refactor dartlang to use atom.dart fs

### PR DESCRIPTION
**Pull request by danrubel**
This removes File, Directory, Entry, and the like from dartlang in favor of identical functionality in atom.dart.

@devoncarew

### CODE REVIEW

File: lib/atom.dart [hunk 1]
```diff
@@ -3,12 +3,13 @@
 // BSD-style license that can be found in the LICENSE file.
 
 /// A Dart wrapper around the [Atom](https://atom.io/docs/api) apis.
-library atom;
+library atom2;
```
Comment (devoncarew) [inline lib/atom.dart]: lgtm; could also drop the library directive altogether

**Comment by devoncarew** [comment]
lgtm!

File: lib/atom.dart (hunk 1 above)
Comment (danrubel) [inline lib/atom.dart]: Good point.

**Comment by devoncarew** [comment]
(landing to resolve some merge conflicts)

**Pull request merged**